lib/utils/path.js

```diff
@@ -1,4 +1,13 @@
 // return the PATH array in a cross-platform way
-const PATH = process.env.PATH || process.env.Path || process.env.path
 const { delimiter } = require('path')
-module.exports = PATH.split(delimiter)
+
+const keys = ['PATH', 'Path', 'path']
+const key = keys.find(k => process.env[k])
```

Comment:
on non-windows, shouldn't only PATH work? meaning, shouldn't be something like `os === 'windows' ? ['Path', 'path'] : ['PATH']`?

Reply:
good call! all three are potentially valid in windows, but outside of windows it should really only be `PATH`

Reply:
will this make testing unneccessarily complicated especially in order to get code coverage?

Reply:
code coverage always needs to be consolidated across multiple runs, to capture "different OS" or "different node versions" - if you have that set up, then this PR won't impact anything; if you don't, then testing is already unnecessarily complicated because you need to scatter istanbul-ignore comments on any code that branches by platform.

Reply:
imo this is a straight forward case and we can ignore coverage for this particular branch. it's pretty simple to see that it's doing the right thing